### DISCUSSION TITLE
Como posso autorizar o dispositivo criado pelo pynubank?

### DISCUSSION THREAD

**Discussion by rafaelneto2** [andreroggeri/pynubank#424] [Q&A]
Após realizar a criação do certificado, chega um e-mail da nubank pedindo que eu autorize o novo dispositivo (PyNubank Client), pelo app. Porém, não sei como fazer isso...

Sem autorizar tenho acesso apenas aos dados do cartão de crédito.

**Comment by andreroggeri** [comment]
Duplicado com a #419
